src/style.css

```diff
@@ -54,8 +54,8 @@ a {
   overflow-y: scroll;
 }
 
-.table-row:not(:hover) .hover-only {
-  visibility: hidden;
+.table-row:not(:hover) .hover-only, .table-cell:not(:hover) .cell-hover-only {
+  display: none !important;
```

Comment:
TOL: 😦

Reply:
Correct. But we don't really have an alternative, unless we make sure all of these have no display attribute set anywhere in their inheritance, since inline always trumps selectors.